tests/Authentication/SessionFilterTest.php

```diff
@@ -12,9 +13,10 @@
 /**
  * @internal
  */
-final class SessionFilterTest extends CIDatabaseTestCase
+final class SessionFilterTest extends CIUnitTestCase
```

Comment:
Shouldn't this extend the new DatabaseTestCase?

Reply:
Done.